docs/lifecycle/lifecycle.rst

```diff
@@ -117,36 +129,129 @@ Board LFN Entry Review
 ++++++++++++++++++++++
 
 It is up to the Board to define its own criteria and process of the Board's LFN
-Entry Review. The TAC recommends the Board make its LFN Entry Review criteria
-and process public and accept design input from the public.
+Entry Review. Such review may include legal, trademark, and license reviews. The 
+TAC recommends the Board make its LFN Entry Review criteria and process public 
+and accept design input from the public.  
+
+Budget Guidance: The TAC recommends that the governing board allocate no funding 
+to Sandbox projects and that those projects cannot use cross-project funding
+buckets.
+
+Quarterly Health Review
+***********************
+
+The TAC should review all projects in the Sandbox state on a quarterly basis. 
+The goal of the quarterly health review is to assist the projects in their 
+participation in LFN and ensure the project's community remains healthy.
+
+Incubation Review
+*****************
+
+The Board and the TAC both review proposals for projects to advance to the 
```

Comment:
2 entities reviewing at the same time => no risk of desync?

consultative review from the TAC submitted to the board for decision is precised just after this statement. why not indicating directly here.